guidelines/coding-guidelines.md

```diff
@@ -11,3 +11,21 @@ For public facing interfaces we favor read-only collections and enumerables. For
 * Avoid using `System.Linq`
 * Avoid using `foreach` over collections that do not have a struct enumerator
 
+## Don't use string interpolation
+
+By using string interpolation it is impossible for logging frameworks that support structure logging to distinguish values within the string. We must use the normal argument based overload so logging frameworks like Serilog can capture structured data.
```

Comment:
> logging frameworks like Serilog can capture structured data

Is that actually correct? Serilog relies on a custom format to actually derive  the structure.  eg

> log.Information("Processed {@Position} in {Elapsed:000} ms.", position, elapsedMs);

in your example 

> logger.Info("File share data bus started. Location: " + basePath);"

Serilog cannot match the variable name `basePath` with its value.

Or am i missing something?